### PR TITLE
Cherry pick from pr123 into dev 025deg jra55do ryf

### PR DESCRIPTION
**Pull request by anton-seaice**
Companion PR to https://github.com/ACCESS-NRI/access-om3-configs/pull/123

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -20,17 +20,18 @@ model: access-om3
...
-    - /g/data/vk83/experiments/inputs/access-om3/mom/grids/vertical/global.025deg/2024.04.04/ocean_vgrid.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/initial_conditions/global.025deg/2020.10.22/ocean_temp_salt.res.nc
-    - /g/data/vk83/experiments/inputs/access-om3/mom/surface_salt_restoring/global.025deg/2020.05.30/salt_sfc_restore.nc
-    - /g/data/vk83/experiments/inputs/access-om3/cice/grids/global.025deg/2024.05.14/grid.nc
-    - /g/data/vk83/experiments/inputs/access-om3/cice/grids/global.025deg/2024.05.14/kmt.nc
-    - /g/data/vk83/experiments/inputs/JRA-55/RYF/v1-4/data
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.025deg/2024.03.26/access-om2-025deg-nomask-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/share/2024.09.16/JRA55do-datm-ESMFmesh.nc
+    - /g/data/vk83/configurations/inputs/access-om3/share/meshes/share/2024.09.16/JRA55do-drof-ESMFmesh.nc
```
Comment (ezhilsabareesh8) [inline config.yaml:26]: Duplication of mesh files

**Review by ezhilsabareesh8 — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (anton-seaice) [reply config.yaml:26]: I think this is correct (i.e. its consistent with :

https://github.com/ACCESS-NRI/access-om3-configs/blob/6ab9a5cc722d9f9540db04d06e67d214d1e400c7/config.yaml#L23-L26

)

Comment (ezhilsabareesh8) [reply config.yaml:26]: Yes that's right.

**Review by ezhilsabareesh8 — APPROVED** [review]
Looks good to me and running fine.

**Pull request merged**